### PR TITLE
drop node dependencies which are unused for building

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by ehelms** [comment]
[test packaging]

**Comment by ekohl** [comment]
Depends on https://github.com/theforeman/foreman/pull/8731

**Comment by evgeni** [comment]
all my changes to core have been merged, so this is now just a simple "call a script and carry on" PR :joy_cat:

**Comment by evgeni** [comment]
one thing this also should help with, is plugin building.
when you build a plugin, it needs to install `foreman-assets`, which takes ~2min to unpack all the content -- less content in there now = faster to unpack \o/

File: debian/bionic/foreman/rules [hunk 1]
```diff
@@ -16,6 +16,7 @@ build:
 	/bin/rm -f bundler.d/test.rb bundler.d/development.rb
 	/bin/sed -i '2i  gem "puma-status"' bundler.d/service.rb
 	BUNDLE_BUILD__SASSC=--disable-march-tune-native BUNDLE_RETRY=5 BUNDLE_JOBS=4 /usr/bin/bundle package
+	/usr/bin/python script/filter-package-json.py
```
Comment (ekohl) [inline debian/bionic/foreman/rules:19]: It's not executable with a shebang?

Comment (evgeni) [reply debian/bionic/foreman/rules:19]: It's not (and the shebang is using py3, which we don't install by default -- the script works with py2 and 3 identically)

**Review by ekohl — APPROVED** [review]

**Pull request merged**